src/components/Stats/StatsCollateralChart/StatsCollateral.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react'
+import { ResponsiveBar } from '@nivo/bar'
+import { colors } from '@static/theme'
+import { Grid, Typography } from '@material-ui/core'
+import useMediaQuery from '@material-ui/core/useMediaQuery'
+import useStyles from './style'
+
+export const StatsCollateralChart: React.FC = () => {
+  const classes = useStyles()
+
+  interface CoinToChart {
+    name: string
+    percent: number
+    color: string
+  }
+
+  const data: CoinToChart[] = [
+    {
+      name: 'xBTC',
+      percent: 20,
+      color: '#6372BE'
+    },
+    {
+      name: 'xETH',
+      percent: 15,
+      color: '#40BFA0'
+    },
+    {
+      name: 'xSOL',
+      percent: 10,
+      color: '#117098'
+    },
+    {
+      name: 'xBNB',
+      percent: 7,
+      color: '#BFB665'
+    },
+
+    {
+      name: 'xFTT',
+      percent: 6.8,
+      color: '#1F70CF'
+    },
+    {
+      name: 'xUSD',
+      percent: 5,
+      color: '#936BC7'
+    },
+    {
+      name: 'xSRM',
+      percent: 5,
+      color: '#39D3F5'
+    },
+    {
+      name: 'xLTC',
+      percent: 4.5,
+      color: '#DADCF1'
+    },
+    {
+      name: 'xAAVE',
+      percent: 3.3,
+      color: '#C76BA2'
+    },
+    {
+      name: 'xDOGE',
+      percent: 2.9,
+      color: '#D49347'
+    },
+    {
+      name: 'xLUNA',
+      percent: 2.2,
+      color: '#DF3C3C'
+    }
+  ]
+
+  const getCoinsName = (data: CoinToChart[]) => {
+    const nameOfCoins: string[] = []
+    data.map(coin => {
+      nameOfCoins.push(coin.name)
+    })
+    return nameOfCoins
+  }
+
+  const getNameAndValue = (data: CoinToChart[]) => {
+    const nameAndValue: {[key: string]: number} = {}
+    for (let i = 0; i <= data.length - 1; i++) {
+      nameAndValue[data[i].name] = data[i].percent
+    }
+    const helperArray: Array<{[key: string]: number}> = []
```

Comment:
I would simplify last three lines to "return [nameAndValue]"

Reply:
or maybe this whole function body to "return data.reduce((object, { name, percent }) => ({
[name]: percent,
...object
}), {})

Reply:
I changed it a little bit, because this data need to return array with object.

    const helperArray = []
    helperArray.push(data.reduce((object, { name, percent }) => ({
      [name]: percent,
      ...object
    }), {}))
    return helperArray